### ISSUE TITLE
Is This Project Dead?

### ISSUE THREAD

**Issue by qwiktune** [witnessmenow/simple-arduino-crypto-display#7]
I wanted to do this project but just noticed people talking about coinmarketcaps API changed and it wont work anymore. As I'm as good as just following step by step instructions on youtube to build it, i'm not sure what that means for a work around. Is this a dead project?

**Comment by piotrwawrzyn** [comment]
I updated the code, take a look: [github.com/piotrwawrzyn/simple-arduino-crypto-display](https://github.com/piotrwawrzyn/simple-arduino-crypto-display)

**Comment by MrPepino** [comment]
Can you tell me which library you use for the display? It either stays black or shows scrambled snow with all the ones I tried. Same display from the readme.

**Comment by piotrwawrzyn** [comment]
Install the newest version of ESP8266 support. [Here](https://github.com/esp8266/Arduino#installing-with-boards-manager) are some simple installation steps if you're not sure.

Also I was just helping another user to set everything up on [thingiverse](https://www.thingiverse.com/thing:2539435/comments), so take a look maybe some of your questions got answered there already.

**Comment by MrPepino** [comment]
Unfortunately not working and stays black screen (no errors). I do see blue led blinking on the board when I plug it in. Thought display might be broken and tried a 2nd display + board but same result.

I reinstalled Arduino and the latest libraries, checked the wires again, etc without improvement. Board support set to ESP8266 NodeMCU 1.0 (ESP-12E Module), Made sure it's not the ESP-12 version because it's NodeMCU V3 (CH340G). The only thing I can think of is that the display driver is not correct. The video says "ESP8266 Oled Driver for SSD1306 display by Daniel and Fabrice" that is either no longer available or renamed to "ESP8266 and ESP32 OLED driver for SSD1306 by ThingPulse and Fabrice" which I currently tried. Are you using the same library?

**Comment by piotrwawrzyn** [comment]
Yes, I think that's the one I'm using. Not sure where the problem lies here. Here is a [zip file](https://mega.nz/file/5aYUSRAA#eCa9GHYcSXAc6aVI3huh9RxH6orHae7nuqKZDMmpcxw) with my libraries. Probably the easiest way to make sure you have the necessary ones.

**Comment by MrPepino** [comment]
Thank you @piotrwawrzyn ! I had been fiddling around till now. It worked immediately the moment I emptied my libraries and added these. Most of these were freshly downloaded earlier besides 3 folders that I didn't have (ACROBOTIC_SSD1306, Adafruit_SSD1306 and Arduino_JSON - I did try Adafruit_SH1106 before).

**Comment by piotrwawrzyn** [comment]
Sure, glad it worked for you! Make sure to download the new version I uploaded a few moments ago, since it fixed a major bug in code.

**Comment by MrPepino** [comment]
@piotrwawrzyn Couldn't post it in your fork so posting it here: it seems the system reboots on a regular base as I see the intro screen + connecting to wifi pop up. It also occurs after replacing my version with different currencies back to your version with the exception of ssid and password. Tried to change the usb cable and charger to be sure. Happens once every 10-15 minutes or so.

**Comment by piotrwawrzyn** [comment]
> @piotrwawrzyn Couldn't post it in your fork so posting it here: it seems the system reboots on a regular base as I see the intro screen + connecting to wifi pop up. It also occurs after replacing my version with different currencies back to your version with the exception of ssid and password. Tried to change the usb cable and charger to be sure. Happens once every 10-15 minutes or so.

Thanks for the feedback @MrPepino, I will definitely look into it in a spare moment.

**Comment by piotrwawrzyn** [comment]
> @piotrwawrzyn Couldn't post it in your fork so posting it here: it seems the system reboots on a regular base as I see the intro screen + connecting to wifi pop up. It also occurs after replacing my version with different currencies back to your version with the exception of ssid and password. Tried to change the usb cable and charger to be sure. Happens once every 10-15 minutes or so.

I think I fixed it. Check out latest update and let me know.

**Comment by piotrwawrzyn** [comment]
For any further issues or questions I enabled issues and discussions in my fork.

**Comment by MrPepino** [comment]
No reboots since yesterday. Nice job and thanks!

ps it's already on my Watchlist so I get notified on updates

**Comment by denstope** [comment]
Hi, I tried your code and there is an error with the crypto.h(says no such file or directory). I am aware that you mentioned something about this on your page, but I don't have any experience with coding. Could you please help me out? Thanks in advance.
